app/api/models/errors/SeBusinessCommonErrors.scala

```diff
@@ -90,3 +90,48 @@ object RuleWrongTpaAmountSubmittedError
       code = "RULE_WRONG_TPA_AMOUNT_SUBMITTED",
       message = "Transition profit acceleration value cannot be submitted without a transition profit value",
       BAD_REQUEST)
+
+object RuleEarlyDataPeriodSummaryNotAcceptedError
+    extends MtdError(
+      code = "RULE_EARLY_DATA_SUBMISSION_NOT_ACCEPTED",
+      message = "Cannot submit data more than 10 days before end of Period.",
+      NOT_FOUND)
+
+object RuleAdvancePeriodSummaryRequiresPeriodEndDateError
+    extends MtdError(
+      code = "RULE_ADVANCE_SUBMISSION_REQUIRES_PERIOD_END_DATE",
+      message = "For advance submission end date must be the end of the period",
+      BAD_REQUEST)
+
+object RulePeriodSummaryEndDateCannotMoveBackwardsError
+    extends MtdError(
+      code = "RULE_SUBMISSION_END_DATE_CANNOT_MOVE_BACKWARDS",
+      message = "Period Summary end date cannot be earlier than existing submission",
+      BAD_REQUEST)
+
+object RuleStartDateNotAlignedWithReportingTypeError
+    extends MtdError(
+      code = "RULE_START_DATE_NOT_ALIGNED_WITH_REPORTING_TYPE",
+      message = "Start date does not align to the reporting type",
+      BAD_REQUEST)
+
+object RuleStartDateNotAlignedToCommencementDateError
+    extends MtdError(
+      code = "RULE_START_DATE_NOT_ALIGNED_TO_COMMENCEMENT_DATE",
+      message = "Start date does not align to the commencement date.",
+      BAD_REQUEST)
+
+object RuleEndDateNotAlignedWithReportingTypeError
+    extends MtdError(code = "RULE_END_DATE_NOT_ALIGNED_WITH_REPORTING_TYPE", message = "End date does not align to the reporting type.", BAD_REQUEST)
+
+object RuleMissingPeriodSummaryDatesError
+    extends MtdError(code = "RULE_MISSING_SUBMISSION_DATES", message = "PeriodSummary start/end date not provided", BAD_REQUEST)
```

Comment:
Change "PeriodSummary start/end date not provided" to "Submission start/end date not provided."

Reply:
done